recipe/meta.yaml

```diff
@@ -40,13 +40,6 @@ test:
   requires:
     - nose
     - mpmath
-  imports:
-    - scipy
-    - scipy.integrate.vode
-    - scipy.linalg._fblas
-    - scipy.linalg._flapack
-    - scipy.sparse.linalg.eigen.arpack._arpack
-    - scipy.sparse.linalg.isolve._iterative
```

Comment:
Dropped as they are all imported in `run_test.py` (checked each one). Also importing them before setting `OPENBLAS_NUM_THREADS` may ignore that setting change.